pkg/cmd/cluster/cluster_attach.go

```diff
@@ -17,7 +17,10 @@ func newCmdClusterAttach(ctx api.Context) *cobra.Command {
 		Short: "Attach the CLI to a cluster",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			manager := ctx.ConfigManager()
+			manager, err := ctx.ConfigManager()
```

Comment:
Why are we sometimes calling the manager `manager` and sometimes `configManager`? Could we normalize the naming?

Reply:
There is no specific reason for this, on the above I just left it as it was called.